From 1559e0e58ac6a5cc4238bd5ca39ec9070013f0ff Mon Sep 17 00:00:00 2001
From: craigbox <craigbox@google.com>
Date: Thu, 31 Mar 2022 14:12:08 +0100
Subject: Update openssf badge (#38189)

* Update openssf badge

Shriram set up a CII badge in 2017, so I thought it was about time to update it.

* Update README.md
---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index b32f0697d3..f9a3846086 100644
--- a/README.md
+++ b/README.md
@@ -1,5 +1,6 @@
 # Istio
 
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1395/badge)](https://bestpractices.coreinfrastructure.org/projects/1395)
 [![Go Report Card](https://goreportcard.com/badge/github.com/istio/istio)](https://goreportcard.com/report/github.com/istio/istio)
 [![GoDoc](https://godoc.org/istio.io/istio?status.svg)](https://godoc.org/istio.io/istio)
 
-- 
2.35.3

